# rosparam load orbit_parameters.yaml [namespace] if don't use namespace /param_name
# Oribit parameters
publish_rate: 10
pos_x: 0.0
pos_y: 0.0
pos_z: 0.0
orientation_x: 0.0
orientation_y: 0.0
orientation_z: 0.0
orientation_w: 1.0
